web/package.json

```diff
@@ -6,7 +6,6 @@
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
     "copy-to-clipboard": "^3.2.0",
-    "dayjs": "^1.8.16",
```

Comment:
You should let dayjs and remove moment, not the other way :stuck_out_tongue:

Reply:
Outdated change

Reply:
Aw got it, I rebase and fucked up this one. Well, nice catch lol